templates/js-cypress/cypress.config.js

```diff
@@ -1,8 +1,5 @@
 import { defineConfig } from 'cypress';
 
 export default defineConfig({
-    e2e: {
-        // to be able to run the tests locally
-        baseUrl: 'https://apify.com',
-    },
+    e2e: {},
```

Comment:
Why did you remove the `baseUrl` ?

Reply:
Because the CLI update should cover this and I did not want to confuse users with default values on too many places.. But now that I think about it might not work with older CLI version so I will add it back